core/rpc/core/src/impl/query.rs

```diff
@@ -1,18 +1,18 @@
-use crate::error::{InnerResult, RpcErrorMessage};
-use crate::rpc_impl::{address_to_script, CURRENT_BLOCK_NUMBER, CURRENT_EPOCH_NUMBER};
-use crate::types::{
-    indexer, indexer_legacy, AssetInfo, Balance, BlockInfo, BurnInfo, GetBalancePayload,
-    GetBalanceResponse, GetBlockInfoPayload, GetSpentTransactionPayload,
-    GetTransactionInfoResponse, IOType, Item, Ownership, QueryTransactionsPayload, Record,
-    StructureType, TransactionInfo, TransactionStatus, TxView,
-};
-use crate::{CkbRpc, MercuryRpcImpl};
+use crate::r#impl::address_to_script;
+use crate::{error::CoreError, InnerResult, MercuryRpcImpl};
 
 use common::utils::parse_address;
 use common::{Context, Order, PaginationRequest, PaginationResponse, Range, SECP256K1};
 use common_logger::tracing_async;
-use core_storage::{DBInfo, Storage};
-use protocol::TransactionWrapper;
+use core_ckb_client::CkbRpc;
+use core_rpc_types::lazy::{CURRENT_BLOCK_NUMBER, CURRENT_EPOCH_NUMBER};
+use core_rpc_types::{
+    indexer, AddressOrLockHash, AssetInfo, Balance, BlockInfo, BurnInfo, GetBalancePayload,
```

Comment:
You should rebase the latest version.

Reply:
The current code has already rebase the `dev` branch.